.taskcat.yml

```diff
@@ -0,0 +1,202 @@
+# cSpell: Disable
+#
+# Repository taskcat configuration.
+#
+# This file defines test cases to run the CloudFormation templates in this
+# repository, which can then be used with the taskcat tool to easily
+# create new stacks.
+#
+# Before using taskcat for first time, you should check and fill in the
+# EKS cluster and XRd settings in the parameters section below with
+# details for your test deployment.
+#
+# You should also check .user_taskcat.yml for more general taskcat settings.
+general:
+  s3_bucket: 917312115843-xrd-quickstart
```

Comment:
Should this be here?  This should be picked up from the user Taskcat template instead?